README.md

```diff
@@ -1,2 +1,88 @@
-# yotas
-Oss's monetisation system
+# Yotas
+
+Oss's reward system
+
+[French](#french)
+
+[English](#english)
+
+# english
+
+Yotas is OSS's reward system, we would like our open source contributors across the organisation to be rewarded for the effort and energy they put into the platform. Each interaction they have on GitHub will give them a certain number of yotas which they can exchange for goodies in an OSS shop, mentoring, ebooks, etc....
+
+## Why?
+Contributing to open source takes some effort, and we would like every contributor to be rewarded for the time and energy they put into helping our community.
+
+## Description
+This reward system is strongly inspired by the correction points and wallet points system applied at Ecole 42. Correction points at Ecole 42 are points that students earn by correcting other students' projects, and lose when they have other students correct their projects. Wallet points are earned when a user contributes to the community, and wallets are directly redeemable for gifts such as 42 logo t-shirts, jacuzzi and other fun rewards.
```

Comment:
We can eliminate this from the description; it will difficult to reference Ecole 42 for a description of their own reward system. Unless we can have an article pointing to it as a reference, then it will quite awesome.

Reply:
done